src/service/user-service.js

```diff
@@ -0,0 +1,56 @@
+import {Op} from "sequelize";
+
+export default class UserService {
+    constructor(userModel, userDataMapper) {
+        this.userModel = userModel;
+        this.userDataMapper = userDataMapper;
+    }
+
+    async add(userDTO) {
+        const user = this.userDataMapper.toDalEntity(userDTO);
+        return (await this.userModel.create(user));
+    };
+
+    async getAutoSuggestUsers(login, limit) {
+        return (await this.userModel.findAll({
```

Comment:
Async/await is a syntax wrapper on promise which makes async code more clear.
The main point about asyn/await syntax is that it allows to deal with async action in synchronous way:
```
const users = await this.userModel.findAll({...});
return users.map(user => this.userDataMapper.toDomain(user));
```
And it is also needed to catch and rethrow  errors on a service layer

Please update other places too